python/ray/tune/syncer.py

```diff
@@ -440,6 +440,8 @@ def get_node_syncer(local_dir, remote_dir=None, sync_function=None):
         return _syncers[key]
     elif not remote_dir or sync_function is False:
         sync_client = NOOP
+    elif local_dir == remote_dir:
```

Comment:
@richardliaw Does this change look good to you? This is to make the tune CLI work locally on windows (if there is no rsync available, e.g. on windows). Is that valueable or should we try to get a platform independent way to sync the results?

Reply:
Unfortunately, I think this would essentially disable all our p2p syncing (for all platforms). `local_dir` and `remote_dir` don't include ip addresses for nodes. For most cases, they are equal. Can we just rely on setting `TUNE_DISABLE_AUTO_CALLBACK_SYNCER ` to avoid windows rsync issue?

Why all unit tests pass is eluding me. @krfricke thoughts?

Reply:
OK, I will revert this and  will try to enable `TUNE_DISABLE_AUTO_CALLBACK_SYNCER` to avoid windows calling into rsync. Thanks for the review.

Reply:
Btw the reason why this probably still works for unit testing is that we don't have proper sync testing in place here. Release tests would probably fail with this test.
With our new local multinode testing we should be able to test these things in the near future.